### ISSUE TITLE
Add release files to release section on GitHub

### ISSUE THREAD

**Issue by vbresan** [tughi/aggregator-android#88]
I am using old Aggregator version 2.22 obtained from Google Play Store. It has package name com.tughi.aggregator
That package is not being updated any more as new versions are published under package name com.tughi.aggregator.next
Meanwhile, on f-droid all updates are under the original package name: com.tughi.aggregator

I have tried to install the updated version from f-droid. The installation has failed as the package is signed with a different key.

I would like to install the new version of Aggregator with the same package name, as I don't want to lose my settings and feed channels history. 

It would be perfect if you could:
1) updated com.tughi.aggregator version on Play Store
2) if 1. is not possible than add release files signed with original keys to release section here on GitHub

Many thanks for your app!

**Comment by tughi** [comment]
I intend to replace the old app at one point, but I don't have the time to work on a proper app migration now.

**Comment by vbresan** [comment]
@tughi In which programming language is the old app written?

**Comment by tughi** [comment]
Old app was written in Java. But the issue is that the required packages to build the app aren't provided by Google anymore...

**Comment by vbresan** [comment]
@tughi I am not a typical app user and I prefer the old version as it is. My motivation to switch to the new version was due to temporary problems reddit had when I opened this issue almost a year ago. I am still using the old version.

Since the old version is my preferred (non-cloud) RSS reader, I would be interested in seeing it open-sourced in any case. I am an Android Java developer, and I might be able to contribute if future issues arise.
